kernel/driver.cc

```diff
@@ -268,7 +269,10 @@ int main(int argc, char **argv)
 		printf("    -A\n");
 		printf("        will call abort() at the end of the script. for debugging\n");
 		printf("\n");
-		printf("    -D <header_id>[:<filename>]\n");
+		printf("    -D <macro>[=<value>]\n");
+		printf("        set the specified Verilog define (via \"read -define\")\n");
+		printf("\n");
+		printf("    -U <header_id>[:<filename>]\n");
```

Comment:
`-U` seems like the inverse of `-D`. What about `-P` instead?

Reply:
SGTM. Changed it accordingly.